packages/core/docs/general/installation.md

```diff
@@ -1,48 +1,37 @@
 # Installation
 
-## Using CLI
+## Using Vue Storefront CLI
 
-If you want to get started with Vue Storefront, the easiest way to do this is to set up your project through our CLI tool. It can be installed globally through NPM:
+The easiest way to get started with Vue Storefront is to set up your project using our CLI. You can run it using the `npx` command:
 
-<code-group>
-<code-block title="YARN">
 ```bash
-yarn global add @vue-storefront/cli
+# Run Vue Storefront CLI
+npx @vue-storefront/cli init
 ```
-</code-block>
-
-<code-block title="NPM">
-```bash
-npm i -g @vue-storefront/cli
-```
-</code-block>
-</code-group>
-
-Once installed, you can use Vue Storefront CLI to set up the project with:
-
-```bash
-vsf init <project_name>
-```
-
-Then you will be asked about the backend platform you wish to use. Once you choose it, the CLI will create the project files in the `<project_name>` directory.
+Enter the name of the project and select the backend platform you wish to use.
 
 <center>
-  <img src="../images/cli.png" alt="vue storefront cli" />
+  <img src="../images/cli.jpg" alt="Vue Storefront CLI" />
 </center>
 
-The only thing that's left before you can start developing is to install the dependencies:
+Once selected, the CLI creates the project files in the directory matching your project name. The only thing left is to go to this directory in the terminal and install the dependencies:
 
 ```bash
-cd <project_name> && yarn
+# Go to project folder
+cd <project_name>
+
+# Install required dependencies
+yarn install
```

Comment:
you're assuming that someone has yarn while previously only mention npm (`npx`)

Reply:
Nice catch. I added the `Prerequisites` section to this document mentioning Node 10+ and Yarn 1